pysoa/client/client.py

```diff
@@ -340,12 +342,27 @@ def call_actions(self, service_name, actions, expansions=None, raise_action_erro
         if timeout:
             kwargs['message_expiry_in_seconds'] = timeout
 
-        request_id = self.send_request(service_name, actions, **kwargs)
-        # Dump everything from the generator. There should only be one response.
+        expected_request_id = self.send_request(service_name, actions, **kwargs)
```

Comment:
So, you changed this to `expected_request_id`, and then changed below from `response_id` to `request_id`. I'm not dead set against that, but I'm not sure it makes sense. The value returned from `send_request` _is_ the request ID, and the value returned from `get_all_responses` _is_ the response ID (and the two are supposed to match). If it were me, I'd leave it `request_id` and `response_id`, but I won't hold it up.